examples/4337-passkeys/package.json

```diff
@@ -6,21 +6,22 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --fix",
```

Comment:
now this command will lint and fix fixable warnings/errors

Reply:
I removed this a while back, because this causes CI to fix lints instead of error on them.

To fix, I run `npm run lint -- --fix`.

Reply:
I pushed a separate script "lint:fix" that runs the above command.

On a side note, I think there's something incorrectly configured, because I never had to run these commands when working on a project. Usually my IDE would pick up all the formatting settings and execute them automatically.

From prettier logs:
```
["INFO" - 11:00:13 AM] Extension Name: esbenp.prettier-vscode.
["INFO" - 11:00:13 AM] Extension Version: 10.4.0.
["INFO" - 11:00:13 AM] Using config file at /home/mmv/Projects/Safe/safe-modules/.prettierrc
["INFO" - 11:00:23 AM] Using config file at /home/mmv/Projects/Safe/safe-modules/.prettierrc
```

Looks like it uses the global prettier file and not the nested one.